docs/en/setup/configuration/process_discovery/overview.md

```diff
@@ -12,4 +12,4 @@ After the process upload is completed, the other modules could perform more oper
 ## Process Detector
 
 Process Detector is used to detect the process from the VM with the different environment:
-1. [Linux Process Detector](./linux.md)
\ No newline at end of file
+1. [Linux Process Scanner](./scanner.md)
```

Comment:
What is the other options? `Container` or `Kubenetes`?

Reply:
Is `Kubernetes`.